BedrockCore.cpp

```diff
@@ -75,6 +75,22 @@ void BedrockCore::prePeekCommand(unique_ptr<BedrockCommand>& command) {
     try {
         try {
             SDEBUG("prePeeking at '" << request.methodLine << "' with priority: " << command->priority);
+
+            // Regularize case
```

Comment:
This comment does not explain why we are doing this, it just explains what the code is doing

Reply:
This is the same comment that lives in `peek()` in auth, I was just pulling the same logic from there.